### PR TITLE
1341: Add release notes for v2.5.0

### PR DESCRIPTION
**Pull request by benedeki**
* Add release notes for v2.5.0
* Description of couple of configuration options

### CODE REVIEW

File: _posts/2020-05-15-release-2.5.0.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+excerpt_separator: <!--more-->
+---
+
+As of 15/05 2020 the new version 2.5.0 is out
+<!--more-->
+
+## Standardization & Conformance Improvements
+
+- [#1294]({{ site.github.issues_url }}/1294) **Record ID generation:** `enceladus_record_id` column is added in Enceladus (Standardization/Conformance) if it does not exist. By default it contains UUID for each record, but the implementation is overridable with `-Denceladus.recordId.generation.strategy={uuid|stableHashId|none}` 
+- [#1322]({{ site.github.issues_url }}/1322) **\_INFO file validation:** Added `control.info.validation` configuration option for \_INFO file validation which can have the following values: `strict` will fail the job on failed validation, `warning` will display the error as warning, but will not fail the job and `none` will ignore the result of the validation. Providing an incorrect value will fail the job with a RuntimeException
```
Comment (Zejnilovic) [inline _posts/2020-05-15-release-2.5.0.md:14]: maybe speak about raw and source checkpoints that are mandatory.

File: _docs/2.0.0/usage/config.md [hunk 2]
```diff
@@ -33,12 +33,16 @@ spark-submit --conf "spark.driver.extraJavaOptions= -Dkey1=value1 -Dkey2=value2"
...
+|                                          |  `warning` (default)                 | A warning message will be displayed on failed validation, but the job will go on. | 
+|                                          |  `none`                              | No validation is done. |
+| `enceladus.recordId.generation.strategy` |  `uuid` (default)                    | `enceladus_record_id` column will be added and will contain a UUID `String` for each row. |
+|                                          |  `stableHashId`                      | `enceladus_record_id` column will be added and populated with an always-the-same `Int` hash (Murmur3-based, for testing). |
+|                                          |  `none`                              | no column will be added to the output. |
+| `menas.rest.uri`                         | string with URLs                    | Comma-separated list of URLs where Menas will be looked for. E.g.: `http://example.com/menas1,http://domain.com:8080/menas2` |
+| `timezone`                               | string with any valid time zone name | The time zone the Spark application will operate in. Strongly recommended to keep it to default _UTC_ |
```
[the remark this answers is not in the document]
Comment (benedeki) [reply _docs/2.0.0/usage/config.md:45]: Suggest a better format 😉

File: _posts/2020-05-15-release-2.5.0.md (hunk 1 above)
Comment (benedeki) [reply _posts/2020-05-15-release-2.5.0.md:14]: Should not be in release notes IMO. Should be in documentation and referenced here.
Will try to put something together.

File: _docs/2.0.0/usage/config.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Zejnilovic) [reply _docs/2.0.0/usage/config.md:45]: the suggestion was too bit. Here is it's PR to merge into this PR #1343

[the remark this answers is not in the document]
Comment (yruslan) [reply _docs/2.0.0/usage/config.md:45]: Sorry, I've approved that PR without knowing it is part of a debate. 😄

[the remark this answers is not in the document]
Comment (Zejnilovic) [reply _docs/2.0.0/usage/config.md:45]: It is fine. This is a non-standard way of doing things but I could not come up with a better one.

File: _docs/2.0.0/usage.md [hunk 3]
```diff
@@ -9,8 +9,9 @@ redirect_from: /docs/usage
 Usage
 =====
 
-* [Schema]({{ site.baseurl }}/docs/{{ page.version }}/usage/schema)
 * [Configuration]({{ site.baseurl }}/docs/{{ page.version }}/usage/config)
+* [\_INFO file]({{ site.baseurl }}/docs/{{ page.version }}/usage/info-file)
```
Comment (Zejnilovic) [inline _docs/2.0.0/usage.md:13]: Currently, no _INFO file usage exists.

Comment (benedeki) [reply _docs/2.0.0/usage.md:13]: Ah, forgot to add it. Sorry.

File: _docs/2.0.0/usage/info-file.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+File named `\_INFO` placed within the source directory together with the raw data is a JSON file tracking control measures 
+via [Atum](https://github.com/AbsaOSS/atum). Example how the file should contain can be found
+[in the code](https://github.com/AbsaOSS/enceladus/blob/master/examples/data/input/_INFO).
+
+
+Validation
+-----------
+
+The _\_INFO file_ verification consists on checking that it has an array field of name `checkpoints`. The array has to
```
Comment (Zejnilovic) [inline _docs/2.0.0/usage/info-file.md:31]:
```suggestion
The _\_INFO file_ verification consists of checking that it has an array field of name `checkpoints`. The array has to
```

File: _docs/2.0.0/usage/info-file.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+have at least to objects one named (field `name`) _"Raw"_ and one _"Source"_. Each of them has the have an array field
```
Comment (Zejnilovic) [inline _docs/2.0.0/usage/info-file.md:32]:
```suggestion
have at least two objects one named (field `name`) _"Raw"_ and one _"Source"_. Each of them has to have an array field
```

File: _docs/2.0.0/usage/config.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (benedeki) [reply _docs/2.0.0/usage/config.md:45]: I think it was pretty good way to address the thing. Let's make it our standard 😉

**Pull request merged**